ui/lib/config-ui/addon/components/messages/page/details.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+
+/**
+ * @module Page::MessageDetails
+ * Page::MessageDetails components are used to display a message
+ * @example
+ * ```js
+ * <Page::MessageDetails @message={{this.message}}  />
+ * ```
+ * @param {model} message - message model
+ */
+
+export default class MessageDetails extends Component {
+  @service store;
+  @service router;
+  @service flashMessages;
+
+  @task
```

Comment:
does this need to be a concurrency task? Do we anticipate the backend taking time to delete something and therefore using the `isRunning` value? I suspect the backend can complete this action very quickly and therefor just using a normal ember data action modifier will be sufficient here, no concurrency needed. 

Checkout how we handle `onDelete` in kv or `onDelete` in ldap.

Reply:
Good point! I don't think the backend will take too long for a delete. Let me update this